### PR TITLE
Allow for custom SSH key names.

### PR DESCRIPTION
**Pull request by timmolter**

### CODE REVIEW

File: src/dk/kaspergsm/stormdeploy/userprovided/Configuration.java [hunk 1]
```diff
@@ -28,7 +28,8 @@
 			"region",
 			"memory-monitor",
 			"remote-exec-preconfig",
-			"remote-exec-postconfig"));
+			"remote-exec-postconfig",
+			"ssk-key-name"));
```
Comment (KasperMadsen) [inline src/dk/kaspergsm/stormdeploy/userprovided/Configuration.java]: ssk?

File: README.markdown [hunk 2]
```diff
@@ -27,6 +27,7 @@ mycluster:
     - region "eu-west-1"
     - remote-exec-preconfig {cd ~, echo hey > hey.txt}
     - remote-exec-postconfig {}
+    - ssk-key-name "mySSHKeyName"           # Optional. defaults to "id_rsa"
```
Comment (KasperMadsen) [inline README.markdown]: ssk?

File: conf/configuration_example.yaml [hunk 3]
```diff
@@ -4,11 +4,12 @@
...
-    - m1.medium {ZK, WORKER, MASTER, UI}			# Request service
-    - m1.medium {ZK, WORKER}					# Request service
-    - m1.medium {ZK, WORKER}					# Request service
+    - m1.medium {ZK, WORKER, MASTER, UI}	# Request service
+    - m1.medium {ZK, WORKER}				# Request service
+    - m1.medium {ZK, WORKER}				# Request service
     - m1.medium {WORKER}					# Request service
     - remote-exec-preconfig {cd ~, echo hey > hey.txt}
     - remote-exec-postconfig {}
+    - ssk-key-name "mySSHKeyName"           # Optional. defaults to "id_rsa"
```
Comment (KasperMadsen) [inline conf/configuration_example.yaml]: ssk?

File: src/dk/kaspergsm/stormdeploy/userprovided/Configuration.java [hunk 4]
```diff
@@ -164,6 +165,19 @@ public String getStormRemoteLocation() {
 		return null;
 	}
 	
+	/**
+	 * Get the ssh key pair name
+	 */
+	public String getSSHKeyName() {
+		String sshKeyName = getRawConfigValue("ssk-key-name");
```
Comment (KasperMadsen) [inline src/dk/kaspergsm/stormdeploy/userprovided/Configuration.java]: ssk?

**Comment by KasperMadsen** [comment]
Hi,
Thanks for the pull request. I have looked at the code and it looks good. I wonder why you use ssk-key-name for the configuration? ... What about ssh-key-name?

**Comment by timmolter** [comment]
whoops, that was a typo. Fix pushed.

**Comment by KasperMadsen** [comment]
Thanks :)

**Pull request merged**